platform/darwin/src/MGLOfflineStorage.mm

```diff
@@ -264,6 +264,34 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
     }
 }
 
+#pragma mark Offline merge methods
+
+- (void)addContentesOfFile:(NSString *)filePath withCompletionHandler:(MGLOfflinePacksAdditionCompletionHandler)completion {
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    //NSAssert([fileManager isWritableFileAtPath:filePath], @"The file %@ must be writable.", filePath);
```

Comment:
`NSAssert()` is only called on debug builds and is intended to flag errors in our part in developing the SDK. Use NSException directly for any errors on the application developer’s part, or call the completion handler with an NSError if this error is out of the application developer’s control.